resources/assets/js/components/JobDetails/JobDetails.tsx

```diff
@@ -0,0 +1,734 @@
+                      </p>
+                    </>
+                  }
+                  label="Select a flexible hours option:"
+                  error={errors.flexHours}
+                  touched={touched.flexHours}
+                  value={values.flexHours}
+                >
+                  <Field
+                    id="flexHoursAlways"
+                    name="flexHours"
+                    component={RadioInput}
+                    label="Almost Always"
+                  />
+                  <Field
+                    id="flexHoursFrequently"
+                    name="flexHours"
+                    component={RadioInput}
+                    label="Frequently"
+                  />
+                  <Field
+                    id="flexHoursSometimes"
+                    name="flexHours"
+                    component={RadioInput}
+                    label="Sometimes"
+                  />
+                  <Field
+                    id="flexHoursOccasionally"
+                    name="flexHours"
+                    component={RadioInput}
+                    label="Occasionally"
+                  />
+                  <Field
+                    id="flexHoursNever"
+                    name="flexHours"
+                    component={RadioInput}
+                    label="Almost Never"
+                  />
+                </RadioGroup>
+                <div data-c-alignment="centre" data-c-grid-item="base(1of1)">
+                  <button
+                    data-c-button="solid(c1)"
+                    data-c-dialog-action="open"
+                    data-c-dialog-id="job-details-preview"
+                    data-c-radius="rounded"
+                    type="submit"
+                    disabled={isSubmitting}
+                  >
+                    Next
+                  </button>
+                </div>
+              </Form>
+              <Modal
+                id="job-details-preview"
+                parentElement="#job-details"
+                visible={isModalVisible}
+                onModalConfirm={() => {
+                  console.log("Confirmed");
+                  setIsModalVisible(false);
+                }}
+                onModalCancel={() => {
+                  console.log("Cancelled");
+                  setIsModalVisible(false);
+                }}
+              >
+                <Modal.Header>
+                  <div
+                    data-c-background="c1(100)"
+                    data-c-border="bottom(thin, solid, black)"
+                    data-c-padding="normal"
+                  >
+                    <h5
+                      data-c-colour="white"
+                      data-c-font-size="h4"
+                      id="job-details-preview-title"
+                    >
+                      You&apos;re off to a great start!
+                    </h5>
+                  </div>
+                </Modal.Header>
+                <Modal.Body>
+                  <div
```

Comment:
Extracting the Modal.Body content to a <JobPreviewCard> component might make it clearer what's going on, (and reduce this file size)

Reply:
And allow it to be reused as the _actual_ job card later

Reply:
Cool, I can do that.